### PR TITLE
Fix #20276: Removed autogenerated migration phpdoc

### PR DESCRIPTION
**Pull request by userator**
When renaming a class using automated tools (IDE), the class name in the comment is ignored and remains in its original form, which misleads the developer.

relevant for versions 2.2 and 2.0

| Q             | A
| ------------- | ---
| Is bugfix? | ✔️
|New feature? | ❌
|Breaks BC? | ❌
|Fixed issues |

### CODE REVIEW

**Review by bizley — APPROVED** [review]

**Comment by userator** [comment]
a duplicate declaration of the class name is meaningless because the class name already tells the developer what kind of class it is

**Review by xepozz — APPROVED** [review]

File: framework/db/BaseActiveRecord.php [hunk 1]
```diff
@@ -1783,7 +1783,7 @@ private function isValueDifferent($newValue, $oldValue)
     {
         if (is_array($newValue) && is_array($oldValue)) {
             // Only sort associative arrays
-            $sorter = function(&$array) {
+            $sorter = function (&$array) {
```
Comment (samdark) [inline framework/db/BaseActiveRecord.php:1786]: No need to change it.

File: framework/db/mssql/Schema.php [hunk 2]
```diff
@@ -823,5 +823,4 @@ public function createColumnSchemaBuilder($type, $length = null)
     {
         return Yii::createObject(ColumnSchemaBuilder::className(), [$type, $length, $this->db]);
     }
-
```
Comment (samdark) [inline framework/db/mssql/Schema.php:826]: No need to change it.

File: framework/views/migration.php [hunk 3]
```diff
@@ -18,7 +18,7 @@
 use yii\db\Migration;
 
 /**
```
Comment (samdark) [inline framework/views/migration.php:20]: Let's remove the docblock altogether. No need for it.

**Review by samdark — CHANGES_REQUESTED** [review]

File: framework/db/mssql/Schema.php (hunk 2 above)
Comment (userator) [reply framework/db/mssql/Schema.php]: returned the codestyle to its original form

File: framework/db/BaseActiveRecord.php (hunk 1 above)
Comment (userator) [reply framework/db/BaseActiveRecord.php]: returned the codestyle to its original form

File: framework/views/migration.php (hunk 3 above)
Comment (userator) [reply framework/views/migration.php]: remove docblock lines

**Review by samdark — APPROVED** [review]

**Comment by userator** [comment]
@samdark fixed

**Comment by samdark** [comment]
Thank you!

**Pull request merged**